src/components/Text/Text.tsx

```diff
@@ -1,12 +1,22 @@
-import React from 'react';
-import PropTypes from 'prop-types';
+import * as React from 'react';
 import bem from '../../utils/bem';
 import styles from './Text.scss';
-import { SIZES, CONTEXTS } from '../../constants';
+import { Size, Context } from '../../constants';
+
+interface Props extends React.HTMLAttributes<HTMLElement> {
+    /** Text content */
+    children: React.ReactElement;
```

Comment:
Maybe make sense write here `React.ReactNode`?. The component will be more flexible and can apply null, undefined... I think it is a use case. What do you think?

Reply:
I did not change the way components are, just migrated them. If we allow for no children, we should that and not render empty html elements. I would rather do these improvements in separate PR. Do you want to make a ticket for it?

Reply:
...hmm, for me not obviously why for `Text` we don't able apply to pass null, undefined, false. Some example:
```
<Text>
{isShow && 'some text' }
</Text>
```
What do you think it can be use case?

Reply:
Anyway, `Text` should support simple text node like this `<Text>hello</Text>` I think.

Reply:
Text can be `number`?